modules/library/referencing/src/test/java/org/geotools/referencing/ScriptTest.java

```diff
@@ -268,7 +268,16 @@ public void testMollweide() throws Exception {
     public void testWagnerIV() throws Exception {
         runScript("scripts/WagnerIV.txt");
     }
-    
+
```

Comment:
The header of this file should be updated to have 2015 as the last modified year